Add docs for validation forms with djmoney (#730)
django-money/django-money: README.rst

* Euros should be between 100 and 1000;
* US Dollars should be between 50 and 500;

> Validation with forms
> ---------------------
> 
> When using with the default ModelForm validation, it is expected that you should
> pass amount and currency as two separate fields:

Reviewer: ```suggestion
Constructing form data
----------------------

The default ``ModelForm`` class will use a form field (``djmoney.forms.fields.MoneyField``) that is constructed of two separate fields for amount and currency.

If you need to feed data directly to such a form (for instance if you are writing a test case), then you need to pass amount and currency like this:

```

.. code:: python

        # models.py
        class Product(models.Model):
            price = MoneyField(
                max_digits=14,
                decimal_places=2,
                default_currency='EUR'
            )

        # forms.py
        class ProductForm(ModelForm):
            class Meta:
                model = Product

>                 fields = "__all__"

Reviewer: ```suggestion
                fields = ["price"]
```

>         ProductForm({'price_0': 10, 'price_1': 'EUR'})

Reviewer: ```suggestion
        # tests.py

        # construct the form in your test case
        form = ProductForm({'price_0': 10, 'price_1': 'EUR'})
```
